test/README.md

```diff
@@ -0,0 +1,149 @@
+# Habitat testing
+
+## Running all tests
+
+First, ensure that all Habitat related environment variables are unset:
```

Comment:
Is there instead a way for us to run these tests in a "clean room" environment? I think this could become tedius since the developers are also habitat users